CMakeLists.txt

```diff
@@ -39,13 +39,28 @@ endif()
 set (SRT_VERSION 1.3.1)
 set_version_variables(SRT_VERSION ${SRT_VERSION})
 
+# Set a default build type if none was specified
+set(default_build_type "Release")
+if(EXISTS "${CMAKE_SOURCE_DIR}/.git")
+  set(default_build_type "Debug")
+endif()
```

Comment:
Not sure if building a cloned git repository should have a *Debug* type by default.

Reply:
I'd prefer not to have any checks if the source tree directory has any `.git` directory and anything depending on it. And the default build type should be `Release`, always. Other types should be explicitly specified.

Reply:
Sure, makes sense. I will remove this.

Reply:
This is now fixed.